.spin/packages.yml

```diff
@@ -0,0 +1,3 @@
+- g++
+- unzip
+- zip
```

Comment:
These are also needed for some special scripts, such as updating expectation files.

```suggestion
- zip
- autoconf
- coreutils
- parallel
```